package.json

```diff
@@ -94,6 +94,7 @@
     "@babel/cli": "^7.4.4",
     "@babel/core": "^7.4.5",
     "@babel/node": "^7.4.5",
+    "@babel/plugin-transform-object-assign": "^7.2.0",
```

Comment:
Babel already provides `_extends` as a helper, we may as well use it.

Reply:
Seems reasonable, we should be careful with swapping mergeObject to Object.assign as Object.assign only does a shallow copy where-as mergeObject does a deep merge.

Reply:
Agreed I only copied default value usages that we had during startup, but we should see if there are more at some point.